### PR TITLE
DO NOT MERGE Docs 3571 flow pinpoint

### PR DESCRIPTION
**Pull request by valkyrie69**
New content. Do not merge until December 8th release

### CODE REVIEW

File: modules/ROOT/pages/monitor-connectors.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
+= Monitor Connectors
+
+You can view performance metrics for supported connectors in your application such as what operation is being performed, how long the operations are taking, and whether they are succeeding or failing.
+
+[NOTE]
+This feature requires a Titanium subscription and the Unified Agent.
```
Comment (dukesphere) [inline modules/ROOT/pages/monitor-connectors.adoc]: Re Titanium, but we link out to the subscription page from other pages: https://docs.mulesoft.com/monitoring/logs (see under the title). Do we want uniformity (this way or that) everywhere?

Comment (valkyrie69) [reply modules/ROOT/pages/monitor-connectors.adoc]: Added

File: modules/ROOT/pages/monitor-connectors.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Supported connectors include:
+
+* Salesforce
+* Database
+* HTTP
+* Other
```
[the remark this answers is not in the document]
Comment (valkyrie69) [reply modules/ROOT/pages/monitor-connectors.adoc]: Other is any connector that is not Salesforce, HTTP, or Database. I suppose I could put that.

File: modules/ROOT/pages/monitor-connectors.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+* Requests: Average number of HTTP connector requests, grouped by automatic intervals
+* Response Time: Average response time of HTTP connector requests, grouped by automatic intervals
+* Failures: Average number of HTTP connector requests failures, grouped by automatic intervals
+|*Other* a|
+
+* Requests: Average number of requests of connectors of type other, grouped by automatic intervals
+* Response Time: Average response time of requests of connectors of type other, grouped by automatic intervals
+* Failures: Average number of requests failures of connectors of type other, grouped by automatic intervals
+
+image:monitoring-connector-dashboard.png[]
```
[the remark this answers is not in the document]
Comment (valkyrie69) [reply modules/ROOT/pages/monitor-connectors.adoc]: If it's okay, I would like to address stuff like this when not on a critical release path.  I'll make a story for myself for enhancement type stuff.

**Pull request merged**